Feature/13

ctillemans opened the pull request:
an early PR so you can see where this is at. It is mostly working but the axios post() method is throwing an error. Does the URL I hardcoded in need another route?

davideugenepratt commented on .vscode/launch.json:
```diff
@@ -0,0 +1,17 @@
+{
```
We should probably add a rule to the .gitignore file int he root of the rep `**/.vscode'

I usually create a workspace file and put it in the same folder as the repo root. So if my repo root is in desktop, I create a workspace and save the workspace file on the desktop so that way all my vscode stuff stays out fo the repo. We should still add a gitignore though jus tin case.

davideugenepratt commented:
@ctillemans it looks like something happened to the **backend/.gitignore** file on this branch and now we have the **backend/node_modules** committed as well as a **backend/.env** file which has the credentials. We should get the **backend/.gitignore** file back in there and then remove the **backend/node_modules** folder and the **backend/.env** file. You also probably should go into the unsplash portal and delete those credentials and create new ones to use since now those old ones are visible in a public repository (not likely to be abused since no one is looking at this but it's a best practice just in case).

ctillemans commented:
I think I know what happened with the .env files. I replaced it at one point and added it to be tracked after a git status. It is back in the .gitignore file and I got new credentials from unsplash.
